### PR TITLE
VAN-613 - Updated styles for Need help signing in? button

### PR DESCRIPTION
**Pull request by shafqatfarhan**

### CODE REVIEW

File: src/forgot-password/ForgotPasswordPage.jsx [hunk 1]
```diff
@@ -109,8 +110,15 @@ const ForgotPasswordPage = (props) => {
                   onClick={handleSubmit}
                   onMouseDown={(e) => e.preventDefault()}
                 />
-                <Hyperlink id="forgot-password" className="btn btn-link font-weight-500 text-body" destination={getConfig().LOGIN_ISSUE_SUPPORT_LINK}>
-                  {intl.formatMessage(messages['need.help.sign.in.text'])}
+                <Hyperlink
+                  id="forgot-password"
+                  className="btn btn-link font-weight-500 text-body"
```
Comment (waheedahmed) [inline src/forgot-password/ForgotPasswordPage.jsx:115]: Remove `btn btn-link` and add `ml-4` fixes the underline issue. You can remove the custom styles with this.

**Review by waheedahmed — APPROVED** [review]
One minor suggestion otherwise looks good.

File: src/forgot-password/ForgotPasswordPage.jsx [hunk 2]
```diff
@@ -109,8 +110,15 @@ const ForgotPasswordPage = (props) => {
...
+                  destination={supportUrl}
+                  onClick={e => {
```
Comment (waheedahmed) [inline src/forgot-password/ForgotPasswordPage.jsx:117]: Can we use `target="_blank"` instead?

File: src/forgot-password/ForgotPasswordPage.jsx (hunk 1 above)
Comment (shafqatfarhan) [reply src/forgot-password/ForgotPasswordPage.jsx:115]: sounds good. Will push the change

Comment (shafqatfarhan) [reply src/forgot-password/ForgotPasswordPage.jsx:115]: done

**Pull request merged**